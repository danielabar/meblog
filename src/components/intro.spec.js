import React from "react"
import renderer from "react-test-renderer"

import Intro from "./intro"

describe("Intro", () => {
  it("renders correctly", () => {
    const testRenderer = renderer.create(<Intro />);
    expect(testRenderer.toJSON()).toMatchSnapshot()
  })
})