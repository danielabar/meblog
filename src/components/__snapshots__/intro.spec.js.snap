// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section>
        <p>
          I'm a software engineer with over 20 years experience delivering software solutions with many languages and frameworks including Ruby on Rails, Node.js, React, Ember, Angular, Go, Python, and Java.
        </p>
        <p>
          I have a passion for crafting high quality software and enhancing team effectiveness with detailed PR reviews, integrating linting tools, automated testing, CI/CD, and developing maintainable code for the long term. I lead by example, showing how these practices can be put into action and encouraging others to follow suit.
        </p>
        <p>
          I also continue to hone my skills with ongoing education, and enjoy sharing my experiences through blogging and mentoring.
        </p>
      </section>
    </div>
  </body>,
  "container": <div>
    <section>
      <p>
        I'm a software engineer with over 20 years experience delivering software solutions with many languages and frameworks including Ruby on Rails, Node.js, React, Ember, Angular, Go, Python, and Java.
      </p>
      <p>
        I have a passion for crafting high quality software and enhancing team effectiveness with detailed PR reviews, integrating linting tools, automated testing, CI/CD, and developing maintainable code for the long term. I lead by example, showing how these practices can be put into action and encouraging others to follow suit.
      </p>
      <p>
        I also continue to hone my skills with ongoing education, and enjoy sharing my experiences through blogging and mentoring.
      </p>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
